examples/batched-solver-from-files/doc/kind

```diff
@@ -0,0 +1 @@
+basic
```

Comment:
That's more for later, but at some point, it will make sense to have a `batched` kind for these examples.

Reply:
Similarly, for clarity of user documentation, I think in the future it would be good to add the `builds-on` file to state this is built on top of the other example (or vice versa) in order to link the batched examples together.

Reply:
Maybe I just add a "batched" kind to both of them. Neither really builds on the other.